mix.exs

```diff
@@ -47,25 +51,23 @@ defmodule ImgProxy.MixProject do
       maintainers: ["Brian Muller"],
       licenses: ["MIT"],
       links: %{
-        "GitHub" => "https://github.com/bmuller/imgproxy",
-        "Changelog" => "https://github.com/bmuller/imgproxy/blob/master/CHANGELOG.md",
+        "Changelog" => "https://hexdocs.pm/imgproxy/changelog.html",
+        "GitHub" => @source_url,
         "imgproxy Site" => "https://imgproxy.net"
       }
     ]
   end
 
-  # Run "mix help compile.app" to learn about applications.
   def application do
     [
       extra_applications: [:logger]
     ]
   end
 
-  # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
       {:credo, "~> 1.5", only: [:dev, :test], runtime: false},
-      {:ex_doc, "~> 0.25", only: :dev}
+      {:ex_doc, "~> 0.24.0", only: :dev, runtime: false}
```

Comment:
This would mean 0.25 wouldn't be supported, but I'd like to support 0.25 and beyond.  We could compromise on the previous version suggestion and just say version 0.25 by default but support anything in the 0.* range:
```suggestion
      {:ex_doc, "~> 0.25", only: :dev, runtime: false}
```